lambdaclass/mirra_backend
# Add 'All' targeting strategy

Closes [Champions of Mirra - Targeting Strategy All](https://github.com/lambdaclass/champions_of_mirra/issues/405)

Added new strategy to target all units in one team or another, not all units in the battle.

## Summary of changes

New `choose_targets` function clause with the 'all' type of targeting strategy, it returns all the units on the caster unit's ally team (including the caster unit) or opposite team, depending of the `target_allies` parameter.

## How to test it?

Set the `targeting strategy` for one of your unit character's skills to "all", you can do so in the `skills.json` file, running `Champions.Config.import_skill_config()` to update the running backend. Then run a battle and verify when the modified skill is cast, it targeted the whole team it was meant for. You can check battle.md on how to run battles.

## Checklist
